src/lib/uart_emul.c

```diff
@@ -436,6 +438,8 @@ uart_write(struct uart_softc *sc, int offset, uint8_t value)
 			ttywrite(&sc->tty, value);
 			if (sc->log.ring)
 				ringwrite(&sc->log, value);
+			if (sc->asl)
```

Comment:
below it looks like the `asl` and `log` are exclusive, as a `else if` is used while here it seems as if they can be used together.

Reply:
I don't see that they are exclusive.  What do you mean?

Reply:
sorry, I misread the snippet of code from below. This is fine.